lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -480,17 +482,16 @@ runGhcid
   -> m ()
 runGhcid root chdirToRoot ghciArgs (toList -> packages) mcmd =
   nixShellWithoutPkgs root True chdirToRoot (packageInfoToNamePathMap packages) "ghc" $
-    Just $ unwords $ ghcidExePath : opts -- TODO: Shell escape
+    Just $ fmap (bash . BSU.fromString) $ ghcidExePath : opts
   where
-    opts =
-      [ "-W"
-      , "--outputfile=ghcid-output.txt"
-      ] <> map (\x -> "--reload='" <> x <> "'") reloadFiles
-        <> map (\x -> "--restart='" <> x <> "'") restartFiles
-        <> testCmd
-        <> ["--command='" <> unwords ("ghci" : ghciArgs) <> "'"] -- TODO: Shell escape
-    testCmd = maybeToList (flip fmap mcmd $ \cmd -> "--test='" <> cmd <> "'") -- TODO: Shell escape
-
+    opts = concat
+      [ ["-W"]
+      , ["--outputfile=ghcid-output.txt"]
+      , map (\x -> "--reload=" <> x) reloadFiles
+      , map (\x -> "--restart=" <> x) restartFiles
+      , maybe [] (\cmd -> ["--test=" <> cmd]) mcmd
+      , ["--command=" <> unwords ("ghci" : ghciArgs)]
```

Comment:
unwords seems like something we should be escaping *again*?

Reply:
Although I see there are no quotes, so maybe this is somehow *flattened*? I thought I tried that before and it didn't work.

Reply:
This turns into `$'--command=ghci args'`

Reply:
Maybe we should be breaking `ghciArgs`?